xrpl/models/base_model.py

```diff
@@ -177,7 +177,7 @@ def _from_dict_single_param(
 
         # no more collections (no params expect a Dict)
 
-        if param_type is Any:
+        if param_type is Any:  # type: ignore
```

Comment:
Is this type annotation necessary?
I updated this line (without the type annotation comment) -- it appears to work correctly

https://github.com/XRPLF/xrpl-py/compare/test-branch-builds...ckeshava:xrpl-py:py38-updates?expand=1

Reply:
It's needed to ignore a new error generated by mypy even though it's working code:

```
poetry run mypy --strict --implicit-reexport xrpl snippets
xrpl/models/base_model.py:180: error: Non-overlapping identity check (left operand type: "Type[Any]", right operand type: "<typing special form>")  [comparison-overlap]
Found 1 error in 1 file (checked 232 source files)
Error: Process completed with exit code 1.
```